api/client/keyring.go

```diff
@@ -0,0 +1,73 @@
+package client
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/cosmos/cosmos-sdk/crypto/hd"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"github.com/celestiaorg/celestia-app/v3/app"
+	"github.com/celestiaorg/celestia-app/v3/app/encoding"
+)
+
+// KeyringConfig contains configuration parameters for constructing
+// the node's keyring signer.
+type KeyringConfig struct {
+	KeyName     string
+	BackendName string
+}
+
+func DefaultKeyringConfig() KeyringConfig {
+	return KeyringConfig{
+		KeyName:     "my_celes_key",
+		BackendName: keyring.BackendTest,
+	}
+}
+
+// NewDefaultRing is a helper function for easy keyring initialization. It initializes a keyring
+// with the given config and path. The returned keyring is ready to use and can be used to sign
+// transactions.
+func NewDefaultRing(cfg KeyringConfig, path string) (keyring.Keyring, error) {
```

Comment:
As a follow-up, how about removing keyring management from `celestia-node` entirely? Especially if this client can be accessed from the command line.

Reply:
isn't this why cel-key is a separate utility?

Reply:
If we remove key management from `celestia-node` it no longer will be able to sign tx (and submit blobs). Do you suggest remove submission from node or maybe i misunderstood smth?